syncd/GlobalSwitchId.cpp

```diff
@@ -15,6 +15,10 @@ sai_object_id_t gSwitchId = SAI_NULL_OBJECT_ID;
 
 #endif
 
+#ifdef MDIO_ACCESS_USE_NPU
+sai_object_id_t mdioSwitchId = SAI_NULL_OBJECT_ID;
+#endif
+
```

Comment:
why you suddenly need this instead of gSwitchId? also i strongly advise to not use global objects, this file is added only for thrift support